pkg/minikube/localpath/localpath_test.go

```diff
@@ -61,3 +66,95 @@ func TestHasWindowsDriveLetter(t *testing.T) {
 		}
 	}
 }
+
+func TestMiniPath(t *testing.T) {
+	var testCases = []struct {
+		env, basePath string
+	}{
+		{"/tmp/.minikube", "/tmp/"},
+		{"/tmp/", "/tmp"},
+		{"", homedir.HomeDir()},
+	}
+	for _, tc := range testCases {
+		originalEnv := os.Getenv(MinikubeHome)
```

Comment:
this originalEnv needs to be before the for loop, otherwise on the second iteration it will have the Wrong original value.

Reply:
good catch. Thanks! Fixing